FRW/Services/Router/Router.swift

```diff
@@ -47,6 +47,35 @@ extension Router {
         }
     }
 
+    static func popToRootAndRoute(to target: RouterTarget, animated: Bool = true) {
```

Comment:
Please use `setViewControllers` to reset the navigation stack. 
 You can refer to
 https://github.com/Outblock/FRW-iOS/blob/86bdf99ef3e67141d67316c838ddbdec09815c56/FRW/Services/Router/RouteMap.swift#L142-L149